app/Filament/Resources/ProjectResource.php

```diff
@@ -77,7 +77,7 @@ public static function form(Form $form): Form
                                                     ->label(__('Ticket prefix'))
                                                     ->maxLength(3)
                                                     ->columnSpan(2)
-                                                    ->unique(Project::class, column: 'ticket_prefix')
+                                                    ->unique(Project::class, column: 'ticket_prefix', ignoreRecord: true)
```

Comment:
Please remove: `ignoreRecord` the word.
It shout look like this `->unique(Project::class, column: 'ticket_prefix', true)`